packages/ui-patterns/src/PageHeader/index.tsx

```diff
@@ -0,0 +1,286 @@
+'use client'
+
+import { cva, type VariantProps } from 'class-variance-authority'
+import React from 'react'
+
+import { cn } from 'ui'
+import { Breadcrumb } from 'ui/src/components/shadcn/ui/breadcrumb'
+import { PageContainer } from '../PageContainer'
+
+// ============================================================================
+// Variants
+// ============================================================================
+
+const pageHeaderVariants = cva(['flex flex-col gap-4 w-full'], {
+  variants: {
+    size: {
+      default: 'pt-12',
+      small: 'pt-12',
+      large: 'pt-12',
+      full: 'pt-6',
+    },
+  },
+  defaultVariants: {
+    size: 'default',
+  },
+})
+
+// ============================================================================
+// Root
+// ============================================================================
+
+export type PageHeaderRootProps = React.ComponentProps<'div'> &
+  VariantProps<typeof pageHeaderVariants> & {
+    size?: 'default' | 'small' | 'large' | 'full'
+  }
```

Comment:
What's the purpose of the intersection with size? Would that not be covered by `VariantProps`?